src/main/webapp/app/add-event.jsp

```diff
@@ -2,47 +2,86 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <html>
 <head>
-    <title>Add Event</title>
+    <title>Add/Edit Event</title>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <link rel="stylesheet" href="/resources/demos/style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
 </head>
-<body>
+<body onload="checkFunction()">
 <style>
     textarea {
 
-        height: 20px;
-        width: 200px;
+        height: 50px;
+        width: 400px;
     }
+    #buttons{
+        position: absolute;
+        bottom: 0;
+        width:100%;
+        height:60px;
+
+    }
+    #add,#edit{
+        border: dotted;
+        text:bold;
+        text-align: center;
+    }
```

Comment:
At least some of these styles are reused in other files - please move all styles to CSS file.